README.md

```diff
@@ -46,19 +46,36 @@ Project documentation is available on the Jetty Eclipse website.
 Building
 ========
 
-To build, use:
+[Apache Maven 3.8.0](https://maven.apache.org/) and [OpenJDK](https://adoptium.net/) requirements:
+
+Branch         | Maven Version | Minimum JDK | Recommended JDK
+---------------|---------------|-------------| ---------------
+`jetty-10.0.x` | Maven 3.8.6+  | OpenJDK 11  | OpenJDK 17 (for optional loom and http/3 support)
+`jetty-11.0.x` | Maven 3.8.6+  | OpenJDK 11  | OpenJDK 17 (for optional loom and http/3 support)
+`jetty-12.0.x` | Maven 3.8.6+  | OpenJDK 17  | OpenJDK 17
+
+Full Build with All Tests:
 
 ``` shell
-  mvn clean install
+mvn clean install
 ```
 
-Eclipse Jetty will be built in `jetty-home/target/jetty-home`.
+Fast Build if you need jars and distribution (not running tests, checkstyle, enforcer, license check):
+
+``` shell
+mvn -Pfast clean install
+```
+Optional build tools: 
 
-The first build may take a longer than expected as Maven downloads all the dependencies.
+* [`graphviz`](https://graphviz.org/) - used by asciidoctor in the jetty-documentation build to produce various graphs
+* [`Docker`](https://www.docker.com/) - used to run some integration tests for testing third party integrations
+
+Eclipse Jetty will be built in `jetty-home/target/jetty-home`.
```

Comment:
```suggestion
The `jetty-home` artifact for standalone server usage will be built in `jetty-home/target/jetty-home`.
```

Reply:
If we are going down this path, then we should expand it further.